Dockerfile

```diff
@@ -16,6 +16,11 @@ RUN dnf module -y enable postgresql:12 && \
     dnf install -y go-toolset postgresql git-core diffutils rpm-devel && \
     ln -s /usr/libexec/platform-python /usr/bin/python3
 
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz \
+    -o oc.tar.gz && \
+    tar -C /usr/bin/ -xvzf oc.tar.gz oc && \
+    rm -f oc.tar.gz
+
```

Comment:
On RHEL it's better to install `oc` from rpm:
`dnf install -y openshift-clients --enablerepo=rhocp-4.6-for-rhel-8-x86_64-rpms`

Reply:
I'm planning to try that as well but I wasn't sure if we have correct subscriptions on ci.ext jenkins nodes, so for now I used the tarball